crates/rolldown/tests/esbuild/dce/dce_of_symbol_instances/artifacts.snap

```diff
@@ -1,6 +1,6 @@
 ---
-source: crates/rolldown_testing/src/case/case.rs
-expression: content
+source: crates/rolldown_testing/src/integration_test.rs
+expression: snapshot
```

Comment:
We can get rid of this line from the snapshot by a setting.

Reply:
Yeah. It's quite annoying. I'll take a look.